src/tests/mod.rs

```diff
@@ -163,6 +165,62 @@ fn test_view_call() {
     })
 }
 
+#[test]
+fn test_accurate_storage_on_selfdestruct() {
+    test_utils::run_test(100, |contract|  {
```

Comment:
You should be able to pass `0` here to get no attached deposit. If you need more expressivity (i.e. to have different balance and deposit), you can update the `test_utils::run_test` function to accept more arguments

Reply:
I need the attached deposit for line 171

Reply:
it doesn't seem to be the number of args that's the issue, more so for the context to change throughout the test if necessary. in that case, an update might be to run a closure for individual transactions that will involve custom contexts other than the default.

Reply:
ah fair point. I guess the closure pattern is not ideal here, and really we should make a shortcut for changing the test env